### PR TITLE
add subscribe category and design Show Confirm popup

### PR DESCRIPTION
**Pull request by cdiazj19**

### CODE REVIEW

File: www/app/categories/categories.controller.js [hunk 1]
```diff
@@ -33,6 +33,23 @@
         ctrlScope.selectedCategory = category;
       };
 
+      ctrlScope.showConfirmSubscribe = function() {
+        var confirmPopup = $ionicPopup.confirm({
+          title: 'SmartQuote App',
+          template: '¿Desea subscribirse a la Categoria?'
+        });
+        confirmPopup.then(function(res) {
```
Comment (rfrancoch) [inline www/app/categories/categories.controller.js:41]: what its "res"?

File: www/app/categories/categories.controller.js [hunk 2]
```diff
@@ -33,6 +33,23 @@
...
+          if(res) {
+            apiSubscription.save({ subscription: { id_user: 1, id_category: ctrlScope.selectedCategory.id } });
+          } else { 
+            }
+        });
+      };
+
+      ctrlScope.subscribe = function(user){
```
Comment (rfrancoch) [inline www/app/categories/categories.controller.js:49]: you don need this code, you have this code in the previous function

File: www/app/categories/categories.html [hunk 3]
```diff
@@ -5,7 +5,7 @@
     <ul class="categories-list">
       <li class="item" ng-class="{ 'selected': isSelected(category)}" 
         ng-repeat="category in categories" 
-        ng-click="selectCategory(category)">
```
Comment (rfrancoch) [inline www/app/categories/categories.html:8]: big error!!!!

in the function showConfirmSubscribe you are assuming that category is selected, but you remove that function!

File: www/app/login/login.controller.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
+(function() {
+  'use strict';
+
+  angular
+    .module('smartquote.login')
+    .controller('LoginController', LoginController);
```
Comment (rfrancoch) [inline www/app/login/login.controller.js:6]: inject the controller function in the module

File: www/app/subscription/subscription.services.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
+(function(){
+  'use strict';
+  	
+  	angular
+  	  .module('smartquote.subscription')
+  	  .factory('ApiSubscription', function ($resource){
+  	  	return $resource('/talend/api/subscription/:id_user/:id_category', { id_user: '@user' },{ id_category: '@category' },
```
Comment (rfrancoch) [inline www/app/subscription/subscription.services.js:7]: @id_user, @id_category

File: www/styles/login.css [hunk 6]
```diff
@@ -0,0 +1,43 @@
+.logo-smartquote{
```
Comment (rfrancoch) [inline www/styles/login.css:1]: see some repeating styles, I recomment only one css file smartquote.css

File: www/app/categories/categories.controller.js (hunk 1 above)
Comment (cdiazj19) [inline www/app/categories/categories.controller.js:41]: @rfrancoch, the function has 'res ' as a parameter, which is the response of the pop-up ask

**Pull request merged**